helical/models/uce/model.py

```diff
@@ -70,6 +69,7 @@ def __init__(self, configurer: UCEConfig = default_configurer) -> None:
     def process_data(self, 
                      adata: AnnData, 
                      gene_names: str = "index",
+                     name = "test",
```

Comment:
I think we can change the name of the dataset here from test to something meaningful (It was test by default previously)

Reply:
- I have left it as test for a default
- But when I use it for fine-tuning I pass a name parameter when using a train and validation set